site/content/en/docs/formats/datumaro_binary.md

```diff
@@ -123,6 +132,30 @@ datum import --format datumaro_binary <path/to/dataset> -- --encryption-key <sec
 
 `<secret-key>` is a 50-bytes long base64 encoded string prefixed with `datum-`. It is auto-generated in `<output/dir>/secret_key.txt` when the dataset is exported to DatumaroBinary format with `--encryption` option. You must have a correct `<secret-key>` to import the dataset encrypted by Datumaro.
 
+### Import datasets with multi-processing
+
+Using CLI
+
+```console
+# Import DatumaroBinary format dataset with 4 multi-processing workers
+datum create
+datum import --format datumaro_binary <path/to/dataset> -- --num-workers 4
```

Comment:
I'm not sure, but do we need additional hyphens? (`-- --num-workers 4`)
I hope.. 
`datum import --format datumaro_binary <path/to/dataset> --num-workers 4`

Reply:
This is a global rule from legacy for CLI argument of all dataset formats. It would be good to make a ticket to enhance this rule for better UX.